spec/requests/api/v1/docs/pages_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "api/v1/pages" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:page) { create(:page) }
+  let(:user) { api_secret.user }
+
+  before do
+    user.add_role(:admin)
+
+    allow(FeatureFlag).to receive(:enabled?).with(:api_v1).and_return(true)
+  end
+
+  path "/api/pages" do
+    describe "get to view all pages" do
+      get("show details for all pages") do
+        tags "pages"
+        description(<<-DESCRIBE.strip)
+        This endpoint allows the client to retrieve details for all Page objects.
+        DESCRIBE
+
+        produces "application/json"
+
+        response(200, "successful") do
+          let(:"api-key") { api_secret.secret }
+          add_examples
```

Comment:
Should we not addd a re-useable global schema for the response object to the show and index specs? 

In addition, it looks like the example values for the pages index method is not showing up (however its showing up for the show method).
<img width="728" alt="Screenshot 2023-01-18 at 10 41 58" src="https://user-images.githubusercontent.com/2786819/213124094-f76132f5-6093-45e4-92de-9ad0ee12f581.png">

Reply:
Good catch -- the index was empty because there aren't any pages in a clean test DB. I switched `let` to `let!` and at least now the docs are less empty:

<img width="663" alt="Screen Shot 2023-01-18 at 11 16 00" src="https://user-images.githubusercontent.com/2077/213145859-180da3d3-1286-482a-a794-2a56fd67bedf.png">

I will look at re-usable schema -- that seems like a good idea, but I'm not as familiar with how we've done this elsewhere.

Reply:
> I will look at re-usable schema -- that seems like a good idea, but I'm not as familiar with how we've done this elsewhere.

Here's an [example](https://github.com/forem/forem/pull/18952/files) if it will help, and you can also read through the Response Blocks section [in the post that I created](https://docs.google.com/document/d/132sGPvcScj5EMo3kR0GjEs1ysRtikSHThqLrIk8qjko/edit#heading=h.5ba7h3blbem2). If there's anything I can do to help or if you see that the instructions in the post aren't clear, let me know 😊